cmd/commands/step.go

```diff
@@ -0,0 +1,177 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/gosimple/slug"
+	"github.com/inngest/inngest/inngest"
+	"github.com/inngest/inngest/pkg/cli"
+	"github.com/inngest/inngest/pkg/cli/form"
+	"github.com/inngest/inngest/pkg/cli/form/formutil"
+	"github.com/inngest/inngest/pkg/function"
+	"github.com/inngest/inngest/pkg/scaffold"
+	"github.com/spf13/cobra"
+)
+
+var ()
+
+func NewCmdSteps() *cobra.Command {
+	root := &cobra.Command{
+		Use:    "step",
+		Short:  "",
+		Hidden: true,
```

Comment:
Why the choice to hide it?

Reply:
This was for internal use during the last week